homeassistant/components/sensor/wallallsensor.py

```diff
@@ -0,0 +1,117 @@
+"""
+All sensors  available on a wall all device
+"""
+from datetime import timedelta
+import logging
+import asyncio
+import voluptuous as vol
+from wallall import wallall
+import homeassistant.helpers.config_validation as cv
+from homeassistant.components.sensor import PLATFORM_SCHEMA
+from homeassistant.const import (
+    CONF_ENTITY_NAMESPACE, CONF_MONITORED_CONDITIONS,CONF_HOST, CONF_PORT,
+    STATE_UNKNOWN, ATTR_ATTRIBUTION)
+from homeassistant.helpers.entity import Entity

```

Comment:
Please follow [PEP8](https://home-assistant.io/developers/development_guidelines/) for layout of your imports.